gems/pending/Gemfile

```diff
@@ -23,7 +23,9 @@ gem "httpclient",              "~>2.7.1",           :require => false
 gem "image-inspector-client",  "~>1.0.3",           :require => false
 gem "iniparse",                                     :require => false
 gem "kubeclient",              "=1.2.0",            :require => false
-gem "linux_admin",             "~>0.17.0",          :require => false
+gem "linux_admin",             "~>0.18.0",          :require => false
+gem "sshkey",                  "~>1.8.0",           :require => false
```

Comment:
Yes, gems/pending tests use the gems/pending/Gemfile.  vmdb tests use the toplevel Gemfile which includes the gems/pending/Gemfile.  This is the correct place to put it.